src/server/constants.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+def defaultEnv(key: str, default: str = ""):
+    value = os.getenv(key)
+    if value is None:
+        return default
+    return value
+
+class PATHS:
```

Comment:
Please add a docstring. Also consider improving the name to `DOCKER_PATHS` or something similar to mark explicitly its usage.

Reply:
Właściwie, to mają to być ścieżki do wszelkich zasobów, a nie tylko dockera, więc nazwa jest prawidłowa